src/app/components/top/solicitudes/formNuevaSolicitud.component.ts

```diff
@@ -215,18 +215,41 @@ export class FormNuevaSolicitudComponent implements OnInit {
             if (this.tipoSolicitud === 'entrada' && !this.modelo.solicitud.profesionalOrigen) {
                 this.plex.confirm('Está a punto de guardar una solicitud de entrada sin indicar profesional de origen', '¿Desea continuar?').then((resultado) => {
                     if (resultado) {
-                        this.guardarSolicitud($event);
+                        this.guardarSolicitud();
                     }
                 });
             } else {
-                this.guardarSolicitud($event);
+                this.guardarSolicitud();
             }
         } else {
             this.plex.info('danger', 'Debe completar los datos requeridos');
         }
     }
 
-    guardarSolicitud($event) {
+    guardarSolicitud() {
+        const params: any = {
+            estados: [
+                'auditoria',
+                'pendiente',
+                'ejecucion'
+            ],
+            idPaciente: this.paciente.id,
+            prestacionDestino: this.modelo.solicitud.tipoPrestacion.conceptId
+        };
+        this.servicioPrestacion.getSolicitudes(params).subscribe(resultado => {
```

Comment:
Este bloque se repite en dos componentes. Sugiero traspasarlo al Service de prestación para evitar duplicidad de código. Algo del estilo `this.servicioPrestacion.getSolicitudesEnCurso(idPaciente, prestacionDestino).subscribe(resultado => { ... });`